setup/context/commands.go

```diff
@@ -63,8 +87,21 @@ func Apply(c *Context, force bool) error {
 		}
 	}
 
-	// write empty dockercfg file if it doesn't exist
-	if err := checkDockercfg(c, force); err != nil {
+	var dockercfg string
+	dockercfg_flag, ok := c.Flags["dockercfg"]
+
+	if ok && dockercfg_flag != nil && *dockercfg_flag != "" {
+		fmt.Printf("Detected dockercfg: `%s`\n", *dockercfg_flag)
+		dockercfg = *dockercfg_flag
```

Comment:
Of course. Should I change the case on `vpc_flag` etc. from which I modeled this from as well?
https://github.com/jonathanstallings/layer0/blob/c8d151db6033d76ac354821be1940df9cab087e3/setup/context/commands.go#L60